src/components/app-view/app-view.jsx

```diff
@@ -22,17 +22,25 @@ const AppSidebar = styled(Sidebar)`
 `;
 
 const Content = styled.main`
+    padding: 32px 0;
     min-width: 0;
-    padding: 32px;
-    position: relative;
+    position: absolute;
     background: #fff;
-    width: calc(100% - ${SIDEBAR_WIDTH}px);
-    transition: transform 0.2s cubic-bezier(0.165, 0.84, 0.44, 1);
-    will-change: transform;
+    right: 0;
+    left: 0;
+    height: 100vh;
+    transition: left 0.3s cubic-bezier(0.165, 0.84, 0.44, 1);
+    will-change: left;
+
+    @media screen and (min-width: ${deviceSizes.tablet}px) {
```

Comment:
can we add check for for deviceSizes.tablet that it is number and if so - add px? we can make a separate helper for it.

Reply:
Done. It will be more like a sass-mixin then utility. It depends on "styled-components" so I define it inside app styles.

Reply:
Check last update. Should I use it over al project?